.github/workflows/pr-build.yml

```diff
@@ -26,4 +26,16 @@ jobs:
       - name: Build Docker container
         shell: sh
         run: |
-          docker build --pull --no-cache --force-rm .
+          docker build --pull --no-cache --tag built-manual-image .
+          docker create --name built-manual-container built-manual-image
+          docker cp built-manual-container:/usr/local/apache2/htdocs "$RUNNER_TEMP/htdocs"
+          docker container rm --force --volumes built-manual-container
+          docker image rm --force built-manual-image
```

Comment:
Just occurred to me that this will be a race condition if multiple PR builds end up getting assigned to the same runner. Maybe the container name and image should be unique for the PR?